### PR TITLE
Fixing tests and running in CI - GitLab Actions

### PR DESCRIPTION
**Pull request by sedan07**

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "2.5" ]
```
Comment (graemedavidson) [inline .github/workflows/php.yml:7]: what does this mean/do?

File: composer.json [hunk 2]
```diff
@@ -59,7 +65,7 @@
     },
     "require-dev": {
         "ext-sqlite3": "*",
-        "alt-three/testbench": "^4.0",
+        "alt-three/testbench": "v5.0.0",
```
Comment (graemedavidson) [inline composer.json:68]: required choice to pin to a specific version?

**Review by graemedavidson — APPROVED** [review]

File: composer.json (hunk 2 above)
Comment (sedan07) [reply composer.json:68]: This is our forked version of it with the naming change to match the paths now used in the cachet project.

File: .github/workflows/php.yml (hunk 1 above)
Comment (sedan07) [reply .github/workflows/php.yml:7]: Runs the CI when any PRs are made against the `2.5` branch.

Comment (graemedavidson) [reply .github/workflows/php.yml:7]: ahh makes sense

File: composer.json (hunk 2 above)
Comment (graemedavidson) [reply composer.json:68]: oh ok, so we will be changing with our changes anyway

**Pull request merged**